### DISCUSSION TITLE
Socials in header

### DISCUSSION THREAD

**Discussion by devzvrm** [squidfunk/mkdocs-material#3051] [Ideas]
I haven't seen this brought up - it seems like socials would make a lot of sense in the header as well. Seems like it would be a simple option to implement (maybe using socialHeader). Being new to mkdocs and material - is there a particular reason this isn't an option or seemingly hasn't been discussed?

**Comment by squidfunk** [comment]
Thanks for suggesting. I think this was discussed before, so maybe it's worth searching the issue tracker. I know of [this](https://docs.coregames.com/) installation which has some socials in the header. In general, the footer is an "easier" place to show something for which the authors can define an arbitrary amount because the header is already pretty packed and you would have to think of a way how this would translate to the mobile view, where there are no tabs and there's even less space.

For this reason, moving socials into the header is left as an exercise to the author 😊 Theme extension is always an option. Furthermore, it's always a tough decision what to add as a feature flag or option, because it adds more complexity. Material for MkDocs is already very complex, probably one of the most complex off-the-shelf themes out there.
